SYNC_UPSTREAM.md

```diff
@@ -8,12 +8,12 @@ Upstream Clang, LLVM, and LLD all sit in different git repositories and they
 are almost changed daily. Sometimes a change upstream may affect several
 projects, but it is usually not easy to figure it out from the commit log.
 
-HCC has been configured use git submodules to track these dependencies, but
+HCC has been configured to use git submodules to track these dependencies, but
 there are users who prefer using repo tool. This document has been revised to
 cope with both git submodules and repo.
 
 HCC depends on amd-common LLVM / LLD / Clang, which is a fork maintained by AMD,
-and may contain patches yet upstreamed. amd-common LLVM / LLD / Clang is
+and may contain patches not yet upstreamed. amd-common LLVM / LLD / Clang is
 automatically synchronized with upstream LLVM / LLD / Clang every 4 hours, so
 they are very close to the latest codes upstream.
 
```

Comment:
@aaronenyeshi an paragraph explaining the interaction between 2 different roles in the process would be good:

- one who conducts the upstream sync process: getting all the codes, merges, file the PRs
- one who reviews the sync PRs